tests/conformance_tests.rs

```diff
@@ -107,11 +79,13 @@ mod implementation {
 mod ion_tests {
     use super::*;
 
-    #[test_resources("ion-tests/conformance/null.ion")]
-    #[test_resources("ion-tests/conformance/core/typed_null.ion")]
-    #[test_resources("ion-tests/conformance/core/string_symbol.ion")]
-    #[test_resources("ion-tests/conformance/core/empty_document.ion")]
-    #[test_resources("ion-tests/conformance/core/toplevel_produces.ion")]
+    #[test_resources("ion-tests/conformance/core/*")]
+    #[test_resources("ion-tests/conformance/data_model/annotations.ion")]
+    #[test_resources("ion-tests/conformance/data_model/boolean.ion")]
+    #[test_resources("ion-tests/conformance/data_model/integer.ion")]
+    #[test_resources("ion-tests/conformance/data_model/null.ion")]
+    // No support for half-precision floats yet.
+    // #[test_resources("ion-tests/conformance/data_model/float.ion")]
```

Comment:
🪧 With these changes, the runner now passes all of `core` and most of `data_model`. However, there are several more things to fix before it will pass the remaining directories.